CMSgov/onemac
# Oy2 18845

Story: https://qmacbis.atlassian.net/browse/OY2-18845
Endpoint: https://d3qdzxk7lhkxxg.cloudfront.net/dashboard

### Details

Admin function for soft deletion and display additional info sections with preserved whitespace.

### Changes

-created admin/handlers/softDeleteChangeRequest to mark a given item as state="inactivated" so that it wont display on the submission dashboard
-added formatting to the submission and package details pages so that additional information will preserve any newlines or tabs input by the user

### Test Plan

1. Create or locate a submission item
2. Login to aws lambda dashboard and run the softDeleteChangeRequest lambda with event similar to:
`{
    "transmittalNumber": "SS.XXXX.RXX.XX",
    "type": "waiver",
    "territory": "SS",
    "prependAdditionalInfo": "UPDATED - Per request ...",
    "submittedLastXMonths": 3
}`
3. Verify item no longer appears in submission dashboard
4. Verify from DB that item has state="inactivated" and the summary field has the prepended text.
